### ISSUE TITLE
Add the language attribute to the <html> element

### ISSUE THREAD

**Issue by sandbergja** [pulibrary/lockers_and_study_spaces#86]
### Expected behavior
The <html> element includes a language attribute in the correct format (see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)

### Actual behavior
The <html> element doesn't include the language attribute.

### Impact of this bug

This is a WCAG 2.1 A violation https://www.w3.org/TR/WCAG21/#language-of-page
